ocs_ci/utility/deployment_openshift_logging.py

```diff
@@ -0,0 +1,383 @@
+    except CommandFailed:
+        logger.error(f'The resource is not found')
+        return False
+    return True
+
+
+def create_clusterlogging_catalogsourceconfig(yaml_file, resource_name):
+    """
+    Creation of catalogsourceconfig to enable the
+    cluster-logging Operator on the cluster.
+
+    Args:
+        yaml_file (str): Path to yaml file to create catalog source config for
+                         cluster-logging
+        resource_name (str) : Name of the catalog source config created
+                             for cluster-logging
+
+    Returns:
+        Boolean : True if catalog source config for cluster-logging is created
+                  successfully, false otherwise
+
+    Example:
+        cl_create_catalogsourceconfig(
+        constants.CL_CSC_YAML, 'cluster-logging-operator')
+
+    """
+
+    create_csc = ocp.OCP(
+        kind=constants.CATALOG_SOURCE_CONFIG, namespace='openshift-marketplace'
+    )
+    clusterlogging_csc = ocp.OCP(
+        kind=constants.CATALOG_SOURCE_CONFIG, namespace='openshift-logging'
+    )
+
+    create_csc.create(yaml_file=yaml_file)
+    try:
+        clusterlogging_csc.get(resource_name, out_yaml_format=True)
+        logger.info(f"The CatalogSourceConfig is created Successfully")
+    except CommandFailed:
+        logger.error(f'The resource is not found')
+        return False
+    return True
+
+
+def create_clusterlogging_subscription(yaml_file):
+    """
+    Creation of subscription for clusterlogging to subscribe
+    a namespace to an operator
+
+    Args:
+        yaml_file : Path to yaml file to create subscription for
+                    the namespace
+        resource_name (str) : Name of the subscription
+
+    Returns:
+        subscription (dict): Contains all the details of the subscription.
+
+    Example:
+        cl_create_subscription(yaml_file=constants.CL_SUB_YAML)
+
+    """
+
+    clusterlogging_subscription = ocp.OCP(
+        kind=constants.SUBSCRIPTION, namespace='openshift-logging'
+    )
+    subscription = clusterlogging_subscription.create(
+        yaml_file, out_yaml_format=True
+    )
+    logger.info(subscription)
+    subscription_info = clusterlogging_subscription.get(
+        resource_name='cluster-logging', out_yaml_format=True
+    )
+    if subscription_info:
+        logger.info(f"The Subscription is created successfully")
+    else:
+        logger.error(f"The subscription is not installed properly")
+    return subscription_info
+
+
+def create_instance_in_clusterlogging(sc_name=None):
+    """
+    Creation of instance for clusterlogging that creates PVCs,
+    ElasticSearch, curator fluentd and kibana pods and checks for all
+    the pods and PVCs
+
+    Args:
+        sc_name (str) : Storage class name to create PVCs
+
+    Returns:
+        logging_instance (dict): Contains all detailed information of the
+        instance such as pods that got created, its resources and limits
```

Comment:
If the description of an item such as the return value spans multiple lines, indent the 2nd+ lines in 4 spaces:

```
def example():
    """
    Some description
    
    Returns:
        dict: some dictionary that
            contains some information
    """
```

Check out the docstring section in our [coding guidelines](https://github.com/red-hat-storage/ocs-ci/blob/master/docs/coding_guidelines.md) for more info.